setup.py

```diff
@@ -18,6 +18,8 @@
 setup(
     name='asyncsnmp',
     install_requires=dependencies,
+    setup_requires=["pytest-runner"],
+    tests_require=["pytest"],
```

Comment:
>tests_require [](start = 4, length = 13)

Add pytest into test_deps, and use test_deps here?

Reply:
resolved